### ISSUE TITLE
Migration of metor to 1.0.2.1

### ISSUE THREAD

**Issue by rajpant** [yogiben/meteor-admin#98]
Hi There, 
I had set the admin package and it was working correctly on meteor 1.0 version. when i upgraded meteor to meteor 1.0.2.1 version the admin sector stopped working with previous collection and its working correctly with newly created collection.
Could you please help me out so that It can work correctly with previous collections as well.

Thanks,

**Comment by rwatts3** [comment]
Are you receiving any console errors ?

**Comment by rajpant** [comment]
No, it show the loading icon on the page. no error on the console. on the dashboard it show the correct no of total records and on the page it doesn't show the data while it show the pagination listing correctly.

**Comment by rajpant** [comment]
Now I have tried it and found that it's not working due to the dot(.) in collection names. my collection name is "places.detail". On the previous version it was working correct but it's not working on current version, could you please help me out.

**Comment by dcsan** [comment]
having a dot in a collection NAME is really asking for trouble... you should change that anyway. it's like putting a / in a filename.

**Comment by yogiben** [comment]
We had added support for collections as properties

e.g. 

`Collection.Posts`

will follow up

**Comment by mpowaga** [comment]
@rajpant Not sure if you have a dot in collection name (e.g. `detail = new Meteor.Collection('places.detail')`) or you have a global object `places` which holds `detail` object. Only the latter will work.

You can also define `collectionObject` property. See [here](https://github.com/yogiben/meteor-admin/issues/12).

**Comment by rajpant** [comment]
@mpowaga  I have the collection name is 'places.detail' andI have used (detail = new Meteor.Collection('places.detail')).
right now i am using in config for admin like:

 detail:{
      icon: 'university',
   
      color: 'yellow',
      tableColumns: [
      ---
---
        
      ]
   },

But it doesn't show me any record in admin panel listing.

Could you please help me out?

**Comment by Maay** [comment]
@mpowaga @yogiben same happen to me, but I don't have any dots in collections.
